building/tooling/test-runners/interface.md

```diff
@@ -119,15 +119,15 @@ The following per-test statuses are valid:
 
 - `pass`: The test passed
 - `fail`: The test failed
-- `error`: The test errored
+- `error`: The test errored - that is, it did not return a value
 
 #### Message
 
 > key: `message`, type: `string`, presence: optional
 
 > version: 2, 3
 
-The per-test `message` key is used to return the results of a failed test. It should be as human-readable as possible. Whatever is written here will be displayed to the student when their test fails. If there is no error message, either set the value to `null` or omit the key entirely. It is also permissible to output test suite output here. The `message` value is not limited in length.
+The per-test `message` key is used to return the results of a test with a `status` of `fail` or `error`. It should be as human-readable as possible. Whatever is written here will be displayed to the student when their test does not pass. If there is no informational message, either set the value to `null` or omit the key entirely. It is also permissible to output test suite output here. The `message` value is not limited in length.
```

Comment:
I don't like "informational message", but it could be nice to avoid the "error" in "error message" here.

Reply:
Maybe we can link it to the status and say something like:

```suggestion
The per-test `message` key is used to return the results of a test with a `status` of `fail` or `error`. It should be as human-readable as possible. Whatever is written here will be displayed to the student when their test does not pass. If there is no test failure or error message, either set the value to `null` or omit the key entirely. It is also permissible to output test suite output here. The `message` value is not limited in length.
```

Reply:
Bump @ee7